services/device-registry-mongodb/src/main/java/org/eclipse/hono/deviceregistry/mongodb/utils/MongoDbDeviceRegistryUtils.java

```diff
@@ -139,4 +142,21 @@ private MongoDbDeviceRegistryUtils() {
                     String.format("Resource [%s] not found.", resourceId)));
         }
     }
+
+    /**
+     * Checks if a provided error is {@link MongoException} with error category {@link ErrorCategory#DUPLICATE_KEY}.
+     *
+     * @param error The exception to check. Other Throwables but {@link MongoException} return {@code false}.
```

Comment:
Something like `@param The error to check.`

Reply:
done